src/PublicKey/RsaOptions.php

```diff
@@ -181,6 +188,28 @@ public function getBinaryOutput()
     }
 
     /**
+     * Get the OPENSSL padding
+     *
+     * @return int
+     */
+    public function getPadding()
+    {
+        return $this->padding;
+    }
+
+    /**
+     * Set the OPENSSL padding
+     *
+     * @param int $padding
```

Comment:
Since the possible values allow `null`, `null` should be an allowed argument as well